python/nano/test/pytorch/tests/test_inference_pipeline_ipex.py

```diff
@@ -259,3 +271,34 @@ def test_get_model_with_method_name(self):
             assert isinstance(model, PytorchIPEXJITModel)
         except:
             pass
+
+    def test_multiple_input_dataloader(self):
+        # will not run this test if torch < 1.10
+        if TORCH_VERSION_LESS_1_10:
+            return
+
+        net = MultipleInputNet()
+        x1 = torch.randn(32, 10)
+        x2 = torch.randn(32, 10)
+        y = torch.randn(32, 1)
+        dataloader = DataLoader(TensorDataset(x1, x2, y), batch_size=1)
+
+        # int8
+        InferenceOptimizer.quantize(net,
+                                    calib_dataloader=dataloader)
+
+        # int8-onnxruntime
+        InferenceOptimizer.quantize(net,
+                                    accelerator="onnxruntime",
+                                    calib_dataloader=dataloader)
+
+        # int8-onnxruntime
+        InferenceOptimizer.trace(net,
+                                 accelerator="onnxruntime",
+                                 input_sample=dataloader)
+
+        # int8-openvino
```

Comment:
here just openvino.
Maybe add an int8-openvino test case.

Reply:
openvino int8 for multiple input case could not run through due to a reason other than dataloader's format, will raise a separate issue and pr for this.